running Mac, Catalina

cd fab_quickstart
pyinstaller cli.py   # fails...
    OSError: Python library not found: libpython3.8.dylib, libpython3.8m.dylib, Python, .Python
        This would mean your Python installation doesn't come with proper library files.
        This usually happens by missing development package, or unsuitable build parameters of Python installation.

        * On Debian/Ubuntu, you would need to install Python development packages
        * apt-get install python3-dev
        * apt-get install python-dev
        * If you're building Python by yourself, please rebuild your Python with `--enable-shared` (or, `--enable-framework` on Darwin)

    python installed via pyenv install 3.8.3, at:
        /Users/val/.pyenv/versions/3.8.3/lib/python3.8

Research found these suggestions, all failed in same way

    This page: https://pyinstaller.readthedocs.io/en/stable/when-things-go-wrong.html
    says set DYLD_LIBRARY_PATH  -- same result
        export DYLD_LIBRARY_PATH=/Users/val/.pyenv/versions/3.8.3/lib/python3.8
        or
        export DYLD_LIBRARY_PATH=/Users/val/.pyenv/versions/3.8.3-dev/lib/python3.8

    pyenv install 3.8-dev  -- same result

    https://stackoverflow.com/questions/43067039/pyinstaller-error-oserror-python-library-not-found-libpython3-4mu-so-1-0-lib
      libpython3.4m.so.1.0 --> /usr/lib -- but can't find mac version

    https://groups.google.com/forum/#!searchin/pyinstaller/pyinstaller$20OSError$3A$20Python$20library$20not$20found/pyinstaller/UJNSXQ6EI4M/KTVYFPVCAQAJ
      brew install pyqt  -- same result

Full Log:

    64 INFO: PyInstaller: 3.6
    64 INFO: Python: 3.8.3
    77 INFO: Platform: macOS-10.15.6-x86_64-i386-64bit
    78 INFO: wrote /Users/val/python/vscode/fab-quickstart/fab_quickstart/cli.spec
    83 INFO: UPX is not available.
    85 INFO: Extending PYTHONPATH with paths
    ['/Users/val/python/vscode/fab-quickstart',
    '/Users/val/python/vscode/fab-quickstart/fab_quickstart']
    85 INFO: checking Analysis
    85 INFO: Building Analysis because Analysis-00.toc is non existent
    85 INFO: Initializing module dependency graph...
    89 INFO: Caching module graph hooks...
    98 INFO: Analyzing base_library.zip ...
    2353 INFO: Processing pre-find module path hook   distutils
    2354 INFO: distutils: retargeting to non-venv dir '/Users/val/.pyenv/versions/3.8.3/lib/python3.8'
    5314 INFO: Caching module dependency graph...
    5441 INFO: running Analysis Analysis-00.toc
    5447 INFO: Analyzing /Users/val/python/vscode/fab-quickstart/fab_quickstart/cli.py
    5864 INFO: Processing module hooks...
    5864 INFO: Loading module hook "hook-pkg_resources.py"...
    6099 INFO: Processing pre-safe import module hook   win32com
    6322 INFO: Excluding import '__main__'
    6323 INFO:   Removing import of __main__ from module pkg_resources
    6324 INFO: Loading module hook "hook-xml.etree.cElementTree.py"...
    6324 INFO: Loading module hook "hook-lib2to3.py"...
    6329 INFO: Loading module hook "hook-encodings.py"...
    6407 INFO: Loading module hook "hook-sysconfig.py"...
    6420 INFO: Loading module hook "hook-xml.py"...
    6502 INFO: Loading module hook "hook-pydoc.py"...
    6502 INFO: Loading module hook "hook-distutils.py"...
    6533 INFO: Looking for ctypes DLLs
    6607 INFO: Analyzing run-time hooks ...
    6611 INFO: Including run-time hook 'pyi_rth_multiprocessing.py'
    6615 INFO: Including run-time hook 'pyi_rth_pkgres.py'
    6632 INFO: Looking for dynamic libraries
    6750 INFO: Looking for eggs
    6750 INFO: Python library not in binary dependencies. Doing additional searching...
    Traceback (most recent call last):
    File "/Users/val/python/vscode/fab-quickstart/venv/bin/pyinstaller", line 8, in <module>
        sys.exit(run())
    File "/Users/val/python/vscode/fab-quickstart/venv/lib/python3.8/site-packages/PyInstaller/__main__.py", line 114, in run
        run_build(pyi_config, spec_file, **vars(args))
    File "/Users/val/python/vscode/fab-quickstart/venv/lib/python3.8/site-packages/PyInstaller/__main__.py", line 65, in run_build
        PyInstaller.building.build_main.main(pyi_config, spec_file, **kwargs)
    File "/Users/val/python/vscode/fab-quickstart/venv/lib/python3.8/site-packages/PyInstaller/building/build_main.py", line 734, in main
        build(specfile, kw.get('distpath'), kw.get('workpath'), kw.get('clean_build'))
    File "/Users/val/python/vscode/fab-quickstart/venv/lib/python3.8/site-packages/PyInstaller/building/build_main.py", line 681, in build
        exec(code, spec_namespace)
    File "/Users/val/python/vscode/fab-quickstart/fab_quickstart/cli.spec", line 6, in <module>
        a = Analysis(['cli.py'],
    File "/Users/val/python/vscode/fab-quickstart/venv/lib/python3.8/site-packages/PyInstaller/building/build_main.py", line 244, in __init__
        self.__postinit__()
    File "/Users/val/python/vscode/fab-quickstart/venv/lib/python3.8/site-packages/PyInstaller/building/datastruct.py", line 160, in __postinit__
        self.assemble()
    File "/Users/val/python/vscode/fab-quickstart/venv/lib/python3.8/site-packages/PyInstaller/building/build_main.py", line 478, in assemble
        self._check_python_library(self.binaries)
    File "/Users/val/python/vscode/fab-quickstart/venv/lib/python3.8/site-packages/PyInstaller/building/build_main.py", line 568, in _check_python_library
        python_lib = bindepend.get_python_library_path()
    File "/Users/val/python/vscode/fab-quickstart/venv/lib/python3.8/site-packages/PyInstaller/depend/bindepend.py", line 945, in get_python_library_path
        raise IOError(msg)
    OSError: Python library not found: libpython3.8m.dylib, .Python, Python, libpython3.8.dylib
        This would mean your Python installation doesn't come with proper library files.
        This usually happens by missing development package, or unsuitable build parameters of Python installation.

        * On Debian/Ubuntu, you would need to install Python development packages
        * apt-get install python3-dev
        * apt-get install python-dev
        * If you're building Python by yourself, please rebuild your Python with `--enable-shared` (or, `--enable-framework` on Darwin)
